Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/Shadow/ESM.azsli

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+ 
+#pragma once
+ 
+#include <Atom/Features/Math/Filter.azsli> // BilinearFilter()
+
+ 
+float SampleESM(const Texture2DArray<float> shadowMap, const SamplerState samp, const float3 uv, const float zReceiver, const float overdark)
```

Comment:
Can you comment on what the overdark is supposed to be?

Reply:
Yeah thats kind of an odd variable name. Changed it to esmExponent